backend/src/main/java/wooteco/retrospective/application/attendance/AttendanceService.java

```diff
@@ -27,21 +32,48 @@ public AttendanceService(AttendanceDao attendanceDao, MemberDao memberDao, TimeD
     public Attendance postAttendance(AttendanceRequest attendanceRequest) {
         validateTime(attendanceRequest);
 
-        Member member = memberDao.findById(attendanceRequest.getMemberId())
-                .orElseThrow(RuntimeException::new);
-        Time time = timeDao.findById(attendanceRequest.getTimeId())
-                .orElseThrow(RuntimeException::new);
-        Attendance attendance = new Attendance(member, time);
+        Attendance attendance = getAttendance(attendanceRequest);
 
         return attendanceDao.insert(attendance);
     }
 
+    @Transactional
+    public void deleteAttendance(AttendanceRequest attendanceRequest) {
+        Attendance attendance = getAttendance(attendanceRequest);
+
+        if (attendanceDao.delete(attendance) != 1) {
+            throw new RuntimeException();
+        }
+    }
+
+    @Transactional(readOnly = true)
+    public Time findTimeById(long timeId) {
+        return timeDao.findById(timeId)
+            .orElseThrow(RuntimeException::new);
+    }
+
+    @Transactional(readOnly = true)
+    public List<Member> findAttendanceByTime(Time time) {
+        return attendanceDao.findByTime(LocalDate.now(), time.getId()).stream()
+            .map(Attendance::getMember)
+            .collect(Collectors.toList());
+    }
+
     private void validateTime(AttendanceRequest attendanceRequest) {
         if (attendanceDao.isExistSameTime(
-                attendanceRequest.getMemberId(),
-                attendanceRequest.getTimeId()
+            LocalDate.now(),
+            attendanceRequest.getMemberId(),
+            attendanceRequest.getTimeId()
         )) {
             throw new IllegalArgumentException("이미 등록된 시간입니다.");
         }
     }
+
+    private Attendance getAttendance(AttendanceRequest attendanceRequest) {
```

Comment:
getAttendance는 새로운 참석자를 "가져온다"라는 느낌이 강한데 막상 소스는 생성하고 있네요. 
```suggestion
    private Attendance createAttendance(AttendanceRequest attendanceRequest) {
```
정도가 어떨까요?